### ISSUE TITLE
Newer solc versions return error when compiling the Factory contract

### ISSUE THREAD

**Issue by ramvi** [ConsenSys/Tokens#33]
```bash
Module build failed: Error: Inline assembly block is not balanced. It leaves 1 item(s) on the stack.
      assembly {
               ^
Spanning multiple lines.

    at Object.module.exports
```

**Comment by simondlr** [comment]
Thanks @ramvi. We will look into this.

**Comment by ramvi** [comment]
@gleim has [updated the code to work with newer versions](https://github.com/gleim/Tokens)

**Comment by simondlr** [comment]
It turns out the inline assembly warning was due to a bug in the solidity compiler that was fixed with 0.4.5. Does this fix your compilation issues?

**Comment by ramvi** [comment]
Yup!
